Source/Diagnostics/ComputeDiagFunctors/BackTransformFunctor.cpp

```diff
@@ -202,6 +228,64 @@ BackTransformFunctor::LorentzTransformZ (amrex::MultiFab& data, amrex::Real gamm
         amrex::Array4< amrex::Real > arr = data[mfi].array();
         amrex::Real clight = PhysConst::c;
         amrex::Real inv_clight = 1.0_rt/clight;
+#ifdef WARPX_DIM_RZ
+        const int n_rz = WarpX::n_rz_azimuthal_modes;
+        const int n_rcomps = WarpX::ncomps;
+        amrex::ParallelFor( tbx,
+            [=] AMREX_GPU_DEVICE (int i, int j, int k)
+            {
+            // arr(x,y,z,comp) has (2*num_rz_modes-1)*10 components
```

Comment:
To reuse the same names as the above variables:
```suggestion
            // arr(x,y,z,comp) has n_rcomps*10 components
```